lib/features/create/Create.js

```diff
@@ -168,7 +168,7 @@ export default function Create(
 
   eventBus.on([ 'create.end', 'create.out', 'create.cleanup' ], function(event) {
     var context = event.context,
-        target = context.target;
+        target = context.target || event.hover;
```

Comment:
Can't we use `event.hover` here in the first place?

Reply:
Shouldn't break anything I guess...